corehq/apps/callcenter/sync_user_case.py

```diff
@@ -83,7 +83,7 @@ def _domain_has_new_fields(domain, field_names):
     return False
 
 
-def sync_user_case(commcare_user, case_type, owner_id, case=None):
+def _sync_user_case(commcare_user, case_type, owner_id, case=None):
```

Comment:
I didn't like having both `sync_usercase` and `sync_user_case` floating around

Reply:
I don't think it's that important, I'm happy with the way it is now.  The main thing I wanted to do was make this a private method and constrain its use to this file.